code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -624,9 +624,9 @@
 	desc = "The Multi-Augmented Severe Operations Networked Resource Integration Gear is an man-portable tank designed for extreme environmental situations. It is excessively bulky, but rated for all but the most atomic of hazards. The specialized armor is surprisingly weak to conventional weaponry. The exo slot can attach most storge bags on to the suit."
 	icon_state = "hardsuit-ancient"
 	item_state = "anc_hardsuit"
-	armor = list("melee" = 10, "bullet" = 5, "laser" = 5, "energy" = 500, "bomb" = 500, "bio" = 500, "rad" = 500, "fire" = 500, "acid" = 500)
+	armor = list("melee" = 25, "bullet" = 25, "laser" = 25, "energy" = 100, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
```

Comment:
Dont buff the armor at all please.
Also the 500+ is for debuffs so you out armor lets say -10 from a suit for fire ect ect.

Reply:
What suits give -400 fire/acid/energy/anything anyways that having 500 resistance is useful? Plus, 6 slowdown is a _lot_ for something with 10 melee res. Pretty sure even stock engi hardsuits have 15 melee.

Reply:
Just keep them above to or above 150.
And adding more armor to the base types makes it to power for what its intended to be used for - Fixing the station and SM dealing

Reply:
For SM-dealing: CEsuit already has 100 rad protect with CE jumpsuit, engi suit has like 75. Since this needs CE access to open (or an emitter but that's **illegal**.), it's about on par with a CE suit that costs 15k(!). Also, don't know what jumpsuits engineers wear that give them -50 rad resist. Plus, hardsuit balance as it is, no hardsuits protect fully from rads by themselves (CEhardsuit 90 + jumpsuit 10 = 100.) For fixing the station: literally a softsuit+welding goggles outclasses this. Normal hardsuits, atmos hardsuits and flashproof plasmaman softsuits are already available at cargo, which have 1 slowdown, same amount of flash protection if you're thinking about welding (2 needed, they give 2.)

Reply:
Legit just keep the vaules at or above 150.
I also dont want any of the melee/laser/bullet armor to be more then 20

Reply:
Can you give a reason other than "I don't want it"?

Reply:
Its not ment to be any good in a real fight. I dont want people abusing it in combat. Thats why.
Its already massively powerful for holding gear and sadly can lead to abuse in that regard - aha Bohx2 ***Which is also intented*** as well as being shock proof / Rad proof making guns that deal eletrical being almost useless, and rad dealing guns pointless